onnx/defs/quantization/defs.cc

```diff
@@ -143,15 +143,22 @@ y = saturate (round (x / y_scale) + y_zero_point)
 ```
 )DOC";
 
+static std::vector<int64_t> default_axes = {1};
+
 ONNX_OPERATOR_SET_SCHEMA(
     DynamicQuantizeLinear,
-    11,
+    13,
     OpSchema()
-        .SetDoc(DynamicQuantizeLinear_ver11_doc)
+        .SetDoc(DynamicQuantizeLinear_ver13_doc)
         .Input(0, "x", "Input tensor", "T1")
         .Output(0, "y", "Quantized output tensor", "T2")
         .Output(1, "y_scale", "Output scale. It's a scalar, which means a per-tensor/layer quantization.", "tensor(float)")
         .Output(2, "y_zero_point", "Output zero point. It's a scalar, which means a per-tensor/layer quantization.", "T2")
+        .Attr(
+            "axes",
```

Comment:
this should be axis and not axes. In the context of this function axis should be singular.